src/components/inputSelect/AposSelectInput.vue

```diff
@@ -56,29 +56,7 @@ export default {
 </script>
 
 <style lang="scss" scoped>
-.apos-input--select {
```

Comment:
Moved this to be able to use it in the pager. This follows with how other input styles were placed in the one file.